src/scripts/chartist-plugin-pointlabels.js

```diff
@@ -29,7 +29,7 @@
               x: data.x + options.labelOffset.x,
               y: data.y + options.labelOffset.y,
               style: 'text-anchor: ' + options.textAnchor
-            }, options.labelClass).text(options.labelInterpolationFnc(data.value));
+            }, options.labelClass).text(options.labelInterpolationFnc(data.value.y += data.value.x ? ', ' + data.value.x : ''));
```

Comment:
@alexstanbury why are you using += here? Should be only + right? Otherwise this would affect the chartist internals as you're changing the data value y.